Formula/brainfuck.rb

```diff
@@ -0,0 +1,30 @@
+class Brainfuck < Formula
+  desc "Interpreter for brainfuck written in C"
+  homepage "https://github.com/fabianishere/brainfuck"
+  url "https://github.com/fabianishere/brainfuck/archive/2.7.1.tar.gz"
+  sha256 "06534de715dbc614f08407000c2ec6d497770069a2d7c84defd421b137313d71"
+
+  head "https://github.com/fabianishere/brainfuck.git"
+
+  option "with-debug", "Compile interpreter with debug support"
+  option "with-examples", "Install brainfuck example programs"
```

Comment:
This shouldn't be an option. How much space do these take up? If not much, I'd just always install them. Otherwise, just drop it.